ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineForkchoiceUpdated.java

```diff
@@ -196,26 +196,10 @@ private boolean isPayloadAttributesValid(
       final Optional<List<Withdrawal>> withdrawals,
       final BlockHeader headBlockHeader) {
 
-    final boolean newTimestampGreaterThanHead =
-        payloadAttributes.getTimestamp() > headBlockHeader.getTimestamp();
-    return newTimestampGreaterThanHead && isWithdrawalsValid(payloadAttributes, withdrawals);
-  }
-
-  private boolean isWithdrawalsValid(
-      final EnginePayloadAttributesParameter payloadAttributes,
-      final Optional<List<Withdrawal>> maybeWithdrawals) {
-    final List<Withdrawal> withdrawals = maybeWithdrawals.orElse(null);
-    return timestampSchedule
-        .getByTimestamp(payloadAttributes.getTimestamp())
-        .map(
-            protocolSpec -> protocolSpec.getWithdrawalsValidator().validateWithdrawals(withdrawals))
-        // TODO Withdrawals this is a quirk of the fact timestampSchedule doesn't fallback to the
-        // previous fork. This might be resolved when
-        // https://github.com/hyperledger/besu/issues/4789 is played
-        // and if we can combine protocolSchedule and timestampSchedule.
-        .orElseGet(
-            () ->
-                new WithdrawalsValidator.ProhibitedWithdrawals().validateWithdrawals(withdrawals));
+    final long newTimestamp = payloadAttributes.getTimestamp();
```

Comment:
nit: maybe a more meaningful var like payloadTimestamp or newPayloadTimestamp?

Reply:
I'll inline the variable